catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -594,26 +586,27 @@ class PoppySegmentedCommand(SegmentedAperture):
                                 in meters of optical path difference (not waves)
     :param dm_config_id: str, name of the section in the config_ini file where information
                          regarding the segmented DM can be found.
-    :param wavelength: wavelength: float, wavelength in nm of the poppy optical system used for
+    :param wavelength: float, wavelength in nm of the poppy optical system used for
                         (extremely oversimplified) focal plane simulations
     :param rotation: float, rotation angle of the hex segmented DM in deg
     :attribute radius: float, half of the flat-to-flat distance of each segment
     :attribute num_terms: int, total number of PTT values on all segments (= 3 x number of segments)
     :attribute dm_command_units: list, the units of the piston, tip, tilt (respecitvely)
                                  values when coming from the DM or DM command
-    :attribute global_coefficients: list of global zernike coefficients in the form
+    :attribute coefficients: list of global zernike coefficients in the form
```

Comment:
You changed the attribute name in the docstring but not the actual attribute itself.